src/components/Proposal/Create/SchemeForms/Validators.ts

```diff
@@ -0,0 +1,49 @@
+import { isHexStrict } from "web3-utils";
+import { isAddress } from "lib/util";
+
+/**
+ * Given a value returns error message in case value is less than 0 or no value provided
+ * @param {any} value
+ */
+// eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types
+export const requireValue = (value: any): string => {
+  let error;
+  if (value === "") {
```

Comment:
value is string or number ?

Reply:
This is kind of a hack. For some reason if there is no value supplied Formik considers the empty value as string, and when the user enters a value of course it's a number.